packages/core/src/Stores/notification-store.js

```diff
@@ -180,6 +190,25 @@ export default class NotificationStore extends BaseStore {
         if (key) this.addNotificationMessage(this.client_notifications[key]);
     }
 
+    addTradeNotification(contract_info = {}) {
+        const { buy_price, contract_id, currency, purchase_time, shortcode, status, underlying } = contract_info;
+        if (this.trade_notifications.some(item => item.contract_id === contract_id)) return;
+        this.trade_notifications.push({
```

Comment:
If `contract_info` was not passed and default value (`{}`) will be used, then we don't get to the `if` statement, but push an object with a lot of `undefined` fields to the `trade_notifications`. 
Is it a real situation, where it can happed?

Reply:
@kate-deriv `contract_info` is always there but I can add a check for this as a failsafe :) TY